App.js

```diff
@@ -2,24 +2,31 @@ import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import Trips from './Components/Trips/Trips';
 import CreateTrip from './Components/CreateTrip/CreateTrip';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import TripPreview from './Components/TripPreview/TripPreview';
+
+export const App = ({ navigation }) => {
+  const Stack = createStackNavigator();
 
-export const App = () => {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <Trips />
-      <CreateTrip />
-    </View>
-  );
+    <NavigationContainer>
+      <Stack.Navigator
+        initialRouteName='Trips'
+        screenOptions={{ gestureEnabled: false }}
+      >
+        <Stack.Screen
+          name='Trips'
+          component={Trips}
+          options={{ title: 'Globe Trotter' }}
+        />
+        <Stack.Screen
+          name='Trip Preview'
+          component={TripPreview}
+        />
+      </Stack.Navigator>
+    </NavigationContainer>
+  )
 }
```

Comment:
Thanks for putting in the work to research the Navigation!

Reply:
My pleasure it was actually pretty cool.  I got confused a couple times so let me know if any questions come up.